src/components/ChatLog.jsx

```diff
@@ -0,0 +1,40 @@
+import './ChatLog.css';
+import PropTypes from 'prop-types';
+import ChatEntry from './ChatEntry';
+
+function ChatLog(props) {
+  const chatMessages = props.entries.map((message) => {
+    return (
+      <div key={message.id} className="chat-log">
+        <ChatEntry
+          id={message.id}
+          sender={message.sender}
+          body={message.body}
+          timeStamp={message.timeStamp}
+          liked={message.liked}
```

Comment:
Since the names of the keys on `entry` are the same as the names your are setting on `ChatEntry` attributes/you are using all of the values in `entry` you could do something something like this to save you a few keystrokes: 

```javascript
  const chatComponents = entries.map((entry) => {
    return(
      <ChatEntry
        {...entry}
        onLikeToggle={onLikeBtnToggle}
        key={entry.id}
      />
    );
  });
```